packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -73,8 +73,148 @@ const useStyles = makeStyles({
     color: theme.alias.color.neutral.brandBackground,
   }),
   rootTint: theme => ({
-    background: theme.global.palette.brand.tint60,
+    backgroundColor: theme.global.palette.brand.tint60,
     color: theme.global.palette.brand.shade40,
+    borderColor: 'none',
+  }),
+  rootFilledDanger: theme => ({
+    backgroundColor: theme.alias.color.red.background3,
+    color: theme.alias.color.neutral.neutralForegroundInvertedAccessible,
+    borderColor: theme.alias.color.red.background3,
+  }),
+  rootOutlineDanger: theme => ({
+    backgroundColor: 'none',
+    color: theme.alias.color.red.foreground3,
+    borderColor: theme.alias.color.red.foreground3,
+  }),
+  rootTintDanger: theme => ({
+    backgroundColor: theme.alias.color.red.background1,
+    color: theme.alias.color.red.foreground1,
+    borderColor: theme.alias.color.red.foreground2,
+  }),
+  rootGhostDanger: theme => ({
+    backgroundColor: 'none',
+    color: theme.alias.color.red.foreground3,
+    border: 'none',
```

Comment:
Try and share common styles, since each time you redeclare styles they will be expanded by our babel plugin to increase bundlesize

Reply:
Done